dm/dm/config/task_cli_args.go

```diff
@@ -37,3 +40,12 @@ func (t *TaskCliArgs) Decode(data []byte) error {
 	err := json.Unmarshal(data, t)
 	return err
 }
+
+// Verify checks if all fields are legal.
+func (t *TaskCliArgs) Verify() error {
+	if t.StartTime == "" {
+		return nil
+	}
+	_, err := time.Parse("2006-01-02 15:04:05", t.StartTime)
+	return terror.Annotate(err, "error while parse start-time, expected in the format '2006-01-02 15:04:05'")
```

Comment:
`2006-01-02 15:04:05` this is too hard for golang user that not familiar with golang 😵, maybe use normal one `yyyy-mm-dd hh-mm-ss` or just leave a example time in error msg

Reply:
"2006-01-02 15:04:05" is just an exmaple for the one not familiar with golang :joy:

Reply:
how about return `"error while parse start-time, expected in the format like '2006-01-02 15:04:05'"`

Reply:
https://github.com/pingcap/tiflow/pull/4261/commits/6a6f97eb4abe22ebffd10ed4b55e2e7187335eba